dashboard-new/src/app/(authenticated)/workflow/[...workflow]/components/Variables.tsx

```diff
@@ -0,0 +1,40 @@
+import { VariableType } from 'littlehorse-client/dist/proto/common_enums'
+import { ThreadSpec, WfRunVariableAccessLevel, WfSpec } from 'littlehorse-client/dist/proto/wf_spec'
+import React, { FC } from 'react'
+
+type VariablesProps = Pick<ThreadSpec, 'variableDefs'>
+
+const variableTypes: { [key in VariableType]: string } = {
+  JSON_OBJ: 'JSON Object',
+  JSON_ARR: 'JSON Array',
+  DOUBLE: 'Double',
+  BOOL: 'Boolean',
+  STR: 'String',
+  INT: 'Integer',
+  BYTES: 'Bytes',
+  UNRECOGNIZED: 'Unrecognized',
+}
+
+const accessLevels: { [key in WfRunVariableAccessLevel]: string } = {
```

Comment:
@eduwercamacaro this reminds me, we may need to think more about Variable Access Levels before `1.0.0`